cwl/src/main/scala/wdl4s/cwl/WorkflowStep.scala

```diff
@@ -51,5 +195,4 @@ object WorkflowStep {
     Array[String] :+:
       Array[WorkflowStepOutput] :+:
       CNil
-
```

Comment:
This whole file might benefit from a Command-Option-L whacking and judicious de-blank-lining